frontend/app/src/components/home/video/ActiveCall.svelte

```diff
@@ -91,39 +91,35 @@
                             } > ${chat.name}`,
                             avatarUrl: client.groupAvatarUrl(chat, app.selectedCommunitySummary),
                             userId: undefined,
-                            messageIndex: chat.videoCallInProgress,
+                            videoCallInProgress: chat.videoCallInProgress,
                         };
                 }
             }
         }
     }
 
-    export async function startOrJoinVideoCall(chat: ChatSummary, join: boolean) {
+    export async function startOrJoinVideoCall(chatId: ChatIdentifier, callType: VideoCallType, join: boolean) {
         if (chat === undefined || iframeContainer === undefined) return;
 
-        const isPublic = !client.isChatPrivate(chat);
-
         try {
             if ($activeVideoCall !== undefined) {
-                confirmSwitchTo = { chat, join };
+                confirmSwitchTo = { chatId, callType, join };
                 return;
             }
 
             // close and threads we have open in the right panel
             ui.filterRightPanelHistory((panel) => panel.kind !== "message_thread_panel");
             removeQueryStringParam("open");
 
-            const callType = isPublic ? "broadcast" : "default";
```

Comment:
This is now passed in to detach the call type from whether the chat is public/private.